README.md

```diff
@@ -151,9 +204,7 @@ export MM_ADMIN_PASSWORD=password
 
```

Comment:
For the dev section I would love to just link to the docs added in https://github.com/mattermost/mattermost-developer-documentation/pull/661. I've just merged that PR and I will take some hours until the changes are visible.

What do you think about a linking to the dev docs?

Reply:
Yes, awesome idea!! I'm 100% happy with that. Could we link there for all the sections that reference development in the READMEs (e.g., Jira, Zoom, etc)?

Reply:
Would this be the correct link: https://developers.mattermost.com/extend/plugins/developer-workflow/?

Reply:
Yes, the section could be used for all plugins. We should just copy&past it to every README we update.

Reply:
https://developers.mattermost.com/extend/plugins/developer-setup/ and https://developers.mattermost.com/extend/plugins/developer-workflow/ are the page that should be linked here.

Reply:
Yes, the section could be used for all plugins. We should just copy&past it to every README we update.

Reply:
Totally agree - a single source is way better and easier to maintain. Thanks for doing those!